UsageDoc/CA_UsageDocument.md

```diff
@@ -241,9 +241,9 @@ Description for the settings in `appSettings.json` file
 
 |S.No| Argument name   |Description  | Is it Mandatory    | Example |
 |--|--|--|--|--|
-| 1 |--packagefilepath   | Path to the package-lock.json file or to the directory where the project is present in case we have multiple package-lock.json files.                                      |Yes  | D:\Clearing Automation|
+| 1 |--packagefilepath   | Path to the package-lock.json file or to the directory where the project is present in case we have multiple package-lock.json files.                                      |Yes  | D:\Clearing Automation, For Docker run /mnt/Input|
```

Comment:
we are giving the mandatory value in the example column . is it okay ?

Reply:
This can be considered as a reference by the user. But in the docker file we have specifies these values for mounting the input

Reply:
reference means the previous value is enough. last discussion u guys mentioned that it is a mandatory value. now saying reference.
simple if it is madat put it in mandatory field otherwise leave it

Reply:
added in mandatory field